### PR TITLE
ISSUE-186(doc): clarify SSO layer and project_name config information

### PR DESCRIPTION
**Pull request by rodriguez-matias**
## What?
Update text:
* SSO layer configuration information about AWS regions.
* How to configurate correctly poyect_name field.

## Why?
* Clarify information about configuration procedure

## References
* 
* closes issue #186, Bug 2 and Bug 4

### CODE REVIEW

**Review by Franr — APPROVED** [review]

File: docs/try-leverage/leverage-project-setup.md [hunk 1]
```diff
@@ -41,7 +41,9 @@ primary_region: us-east-1
 secondary_region: us-west-2
 ...
 ```
-!!! info "We took `us-east-1` as an example, but it might not be the best region for your use case. For more information, you can follow these [guidelines](https://binbash.atlassian.net/wiki/external/2159050753/NGZkZWMzMTA5M2UwNDQ0OTk3MjUwMGY5NjA0YzRjNjk)."
+!!! info "The `project_name` field only accepts lowercase alphanumeric characters and allows hyphens('-'). For instance, valid names could be 'example' or 'leveragedemo' or 'example-demo' "
+
+!!! info "We typically use `us-east-1` and `us-west-2` as our default regions for the majority of our projects. However, please note that these regions may not be the most fitting choice for your specific use case. For detailed guidance, we recommend following these provided [guidelines](https://binbash.atlassian.net/wiki/external/2159050753/NGZkZWMzMTA5M2UwNDQ0OTk3MjUwMGY5NjA0YzRjNjk)."
```
Comment (exequielrafaela) [inline docs/try-leverage/leverage-project-setup.md:46]: @rodriguez-matias Matu I would suggest an small adjustment adding 1ry and 2ry in this sentence. 

> We typically use as 1ry `us-east-1` and 2ry `us-west-2` as our default regions for the majority of our projects

File: docs/try-leverage/management-account.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ $ leverage credentials configure --type BOOTSTRAP --skip-access-keys-setup
 !!! info "More information on [`credentials configure`](/user-guide/leverage-cli/reference/credentials#configure)"
 
 ### SSO layer
-Before working on the SSO layer you have to navigate to the [AWS IAM Identity Center page](https://console.aws.amazon.com/singlesignon/) and enable Single Sign-On (SSO) by clicking on the `Enable` button.
+Before working on the SSO layer you have to navigate to the [AWS IAM Identity Center page](https://console.aws.amazon.com/singlesignon/) and enable Single Sign-On (SSO) by clicking on the `Enable` button. Additionally, to ensure a correct integration, make sure to set the primary region for SSO in AWS console to match the primary region of your Leverage project.
```
Comment (exequielrafaela) [inline docs/try-leverage/management-account.md:116]: @rodriguez-matias Matu here I would add at the end: "e.g. `us-east-1` for both of them."

**Then:**
Additionally, to ensure a correct integration, make sure to set the primary region for SSO in AWS console to match the primary region of your Leverage project. e.g. `us-east-1` for both of them.

**Review by exequielrafaela — CHANGES_REQUESTED** [review]
@rodriguez-matias Matu some small suggestions, in case you find a better version don't hesitate to adjust it as you see fit. Thanks!

**Comment by rodriguez-matias** [comment]
@exequielrafaela  Hi, the requested changes are ready.

**Review by exequielrafaela — APPROVED** [review]

**Pull request merged**